server/tso/allocator_manager.go

```diff
@@ -245,6 +245,37 @@ func (am *AllocatorManager) GetDCLocationInfo(dcLocation string) (DCLocationInfo
 	return infoPtr.clone(), true
 }
 
+// CleanUpDCLocation cleans up certain server's DCLocationInfo
+func (am *AllocatorManager) CleanUpDCLocation() error {
+	am.mu.Lock()
+	defer am.mu.Unlock()
+
+	// Check if there is any dc-location to clean up
+	serverID := am.member.ID()
+	dcLocationKey := am.member.GetDCLocationPath(serverID)
+	if resp, err := kv.
```

Comment:
Could we just try to delete it directly? I think a read before the deleting is unnecessary.